hail/python/hail/methods/impex.py

```diff
@@ -2975,3 +2975,151 @@ def import_avro(paths, *, key=None, intervals=None):
         with DataFileReader(avro_file, DatumReader()) as data_file_reader:
             tr = ir.AvroTableReader(avro.schema.parse(data_file_reader.schema), paths, key, intervals)
     return Table(ir.TableRead(tr))
+
+
+@typecheck(paths=oneof(str, sequenceof(str)),
+          key=table_key_type,
```

Comment:
Can you add one space before every line here? The `key` should line up with the `paths`.